lib/prototype-admin/clear-data.html

```diff
@@ -21,8 +21,9 @@
       <h1 class="govuk-heading-xl">
         Clear data?
       </h1>
+      <p>This will clear data all session data and reload the default session data.
```

Comment:
I think potentially your git history has got a bit mixed up since this issue you resolved is here again.